code/modules/economy/commodity.dm

```diff
@@ -2900,3 +2880,1072 @@
 	price = 21000
 	upperfluc = 20000
 	lowerfluc = -28500
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//  ______   _        _______  _______  _                _______  _______  _______  _        _______ _________ //
+// (  ___ \ ( \      (  ___  )(  ____ \| \    /\        (       )(  ___  )(  ____ )| \    /\(  ____ \\__   __/ //
```

Comment:
can you please drop the ASCII art? It makes it harder to read whats going on

Reply:
no